cirq/contrib/two_qubit_gates/example_test.py

```diff
@@ -0,0 +1,5 @@
+from cirq.contrib.two_qubit_gates import example
+
+
+def test_gate_compilation_example():
+    example.main(samples=10, max_infidelity=0.3, verbose=False)
```

Comment:
Nice job reducing the cost of running the example test.

Reply:
Thanks!

Reply:
Thanks!